mnist/dockerfile/Dockerfile.ksonnet

```diff
@@ -0,0 +1,18 @@
+# This container is for running ksonnet within Kubernetes
```

Comment:
Where in the example does running ksonnet in cluster come into play?

Reply:
in the model-train.yaml.

Reply:
So this is so that we can have an Argo workflow that creates a ksonnet app to deploy the model?

Reply:
yes, the model-train.yaml is a workflow, the ks is run inside the workflow.